app/forms/openid_connect_authorize_form.rb

```diff
@@ -135,5 +136,10 @@ def error_redirect_uri
       state: state
     )
   end
+
+  def validate_privileges
+    return unless loa3_requested? && service_provider.ial != 2
```

Comment:
Another option would be to fix the definition of `ial` in line 113/114 and then do 
```return unless ial > service_provider.ial```

This would require a tweak to line 90 of `app/presenters/openid_connect_user_info_presenter.rb` and maybe elsewhere. At some point, we assumed LOA and IAL levels were identical.

Reply:
Unfortunately there are already entries in the db for identities.ial = 3